hypervisor/include/common/schedule.h

```diff
@@ -90,6 +90,14 @@ struct sched_iorr_control {
 	struct hv_timer tick_timer;
 };
 
+#ifdef CONFIG_SCHED_BVT
```

Comment:
we can remove CONFIG_SCHED_BVT around the definitions. Else there are too much MACROs

Reply:
fine.